test/without_keys/test_parameter_value_generator.py

```diff
@@ -47,19 +47,6 @@ def test_parameter_value_generator(self):
         )
         self.assertIsInstance(new_params, pybamm.ParameterValues)
 
-        chemistry = pybamm.parameter_sets.Chen2020_plating
```

Comment:
`Chen2020_plating` is now `OKane2022` (with some modifications). You can try adding the latter and see if it works.